Makefile

```diff
@@ -636,42 +638,36 @@ run-c-tests: all-c-tests
 	@echo '---- Run C unit-tests: ----'
 	./$(C_UNIT_TEST_BIN)
 	@echo
-	./$(L3_DUMP)
```

Comment:
These lines are deleted because with built-in argument parsing, by just running the tool by itself, we will get a usage error and a hard-failure:

```
Macintosh:[295] $ ./l3_dump.py
usage: l3_dump.py [-h] --log-file <log-file-name> --binary <program-binary> [--verbose] [--debug]
l3_dump.py: error: the following arguments are required: --log-file, --binary

Macintosh:[296] $ echo $?
2
```

This breaks test-execution, so deleted this call.